src/add-and-remove.js

```diff
@@ -0,0 +1,35 @@
+import { indexOf } from "lodash"; // eslint-disable-line
+
+export function removeItem(obj){
+    obj.remove();
+    let existingEntries = JSON.parse(localStorage.getItem('todos'));
+      let deletedItem = obj.id;
+      let entry = existingEntries.find(obj => obj.index == deletedItem);
+      existingEntries.splice(existingEntries.indexOf(entry), 1);
+      localStorage.setItem('todos', JSON.stringify(existingEntries));
```

Comment:
- [ ] Kindly make sure when an item is deleted, the list of todos is resorted correctly by index.
For example, we have one, two, and three todo items in the list, one has an index = 0, two = 1, and three = 2, and we deleted number two on the list (index = 1), the 3rd item should have an index = 1.
_Please take a look at the following pictures and check the elements (IDs) in the devTools!_
![To Do List - Google Chrome 9_24_2021 7_10_50 PM (1)](https://user-images.githubusercontent.com/47625407/134715174-f37610c4-3b81-4752-b141-b1ed65847f9a.png)
![To Do List - Google Chrome 9_24_2021 7_12_00 PM](https://user-images.githubusercontent.com/47625407/134715182-fade736b-3abb-4521-8cf4-1d24989e7440.png)

>Please go ahead and check out the project requirements for this point right [here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Deleting%20a%20task%20should,saved%20in%20local%20storage.).